submission/submission-webapp/gradle.properties

```diff
@@ -1,2 +1,3 @@
 coverage=0.7
 enableFire=true
+enableTask=false
```

Comment:
I think we should name it `enableTaskMode` just to keep it consistent with the actual application's yml